connector/go.mod

```diff
@@ -53,16 +72,6 @@ replace go.opentelemetry.io/collector/confmap => ../confmap
 
 replace go.opentelemetry.io/collector/consumer => ../consumer
 
-replace go.opentelemetry.io/collector/extension => ../extension
-
 replace go.opentelemetry.io/collector/featuregate => ../featuregate
 
 replace go.opentelemetry.io/collector/pdata => ../pdata
-
-replace go.opentelemetry.io/collector/processor => ../processor
-
-replace go.opentelemetry.io/collector/receiver => ../receiver
-
-replace go.opentelemetry.io/collector/exporter => ../exporter
```

Comment:
This is nice 👍

Reply:
Removal of the deprecated `obsreport/obsereporttest` in the next release should give us a huge reduction go.sum in the diff 🎉

Reply:
yes! less interdependencies is exciting